ngraph/frontend/tensorflow/src/op/gather.cpp

```diff
@@ -2,65 +2,46 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <op_table.hpp>
-#include <openvino/opsets/opset8.hpp>
+#include "op_table.hpp"
+#include "openvino/opsets/opset8.hpp"
 
 using namespace std;
 using namespace ov::opset8;
 
-// See .../tensorflow/include/tensorflow/cc/ops/array_ops.h
-// and .../openvino/ngraph/core/include/ngraph/op/gather.hpp
 namespace ov {
 namespace frontend {
 namespace tf {
 namespace op {
 
-OutputVector TranslateGatherOp(const NodeContext& node) {
-    auto ng_input = node.get_ng_input(0), ng_input_indices = node.get_ng_input(1);
-
-    auto ng_axis = ConstructNgNode<Constant>(node.get_name(), element::i64, Shape{}, 0);
-
-    auto gather_op = ConstructNgNode<Gather>(node.get_name(), ng_input, ng_input_indices, ng_axis);
-
-    return {gather_op};
+OutputVector translate_gather_op(const NodeContext& node) {
+    auto ng_input = node.get_input(0);
```

Comment:
please add `TODO: fix this code to use partial shapes, arbitrary axis and batch dims inputs`

Reply:
axis is always 0 for gather v1. 
.get_shape is not called in this code.